apps/vs-code-designer/src/package.json

```diff
@@ -847,7 +847,7 @@
           },
           "azureLogicAppsStandard.autoBinariesInstallation": {
             "type": "boolean",
-            "description": "Start using binaries installation",
+            "description": "Start using binaries installation Test",
```

Comment:
We don't need "Test", correct?

```suggestion
            "description": "Start using binaries installation",
```

Reply:
Yes, no need for the TEST. Sorry I just added for the PR to be able to mark where I wanted you to review the string. 
Here it is how its going to look like
<img width="1397" alt="Screenshot 2023-10-30 at 11 01 40 AM" src="https://github.com/Azure/LogicAppsUX/assets/102700317/f10a8c63-cda4-4771-84b0-3cc9aefb948e">